style.css

```diff
@@ -0,0 +1,60 @@
+body{
+background-color:rgb(238, 230, 230);
+}
+h1{
+color:red;
+text-align: center;
+}
+.table{
+box-shadow: 0 10px 30px rgba(0,0,0,0.2);
+width:400px;
+background-color:white;
+box-sizing: border-box;
+border-collapse: collapse;
+margin: 0 auto;
+}
+.div_bottom{
+font-size: 10pt;
+height:20px;
+color:grey;
+display:inline-block;
+padding: 5px 1px 5px 10px;
+}
+
+.div_first{
+color:grey;
+border-bottom: 1px solid grey;
+padding: 10px 1px 10px 10px;
+}
+.div_second{
+text-decoration:line-through;
+color:grey;
+border-bottom: 1px solid grey;
+padding: 10px 1px 10px 10px;
+}
+
+.div_3{
+border-bottom: 1px solid grey;
+padding: 10px 1px 10px 10px;
+}
+.div_bottom_1{
+    display:inline-block;
```

Comment:
you can create separate classes to include common styles and add them to elements.

Reply:
changed